libs/ui-lib/lib/ocm/components/clusterWizard/disconnected/OptionalConfigurationsStep.tsx

```diff
@@ -121,7 +121,7 @@ const OptionalConfigurationsStep = () => {
               name: `disconnected-cluster_infra-env`,
               pullSecret: values.pullSecret || pullSecret,
               clusterId: cluster.id,
-              openshiftVersion: '4.19',
+              openshiftVersion: '4.20',
```

Comment:
Any other option than hardcoding? May be get the latest stable OCP release  and extract major.minor version from it?